charts/openstack-cloud-controller-manager/values.yaml

```diff
@@ -0,0 +1,76 @@
+image:
+  repository: docker.io/k8scloudprovider/openstack-cloud-controller-manager
+  tag: ""
+
+resources: {}
+# resources:
+#   requests:
+#     cpu: 200m
+
+livenessProbe: {}
+# livenessProbe:
+#   httpGet:
+#    path: /metrics
+#    port: 10258
+readinessProbe: {}
+
+nodeSelector: []
+# nodeSelector:
+#   node-role.kubernetes.io/controlplane: "true"
+
+tolerations: []
+# tolerations:
+#   - key: node.cloudprovider.kubernetes.io/uninitialized
+#     value: "true"
+#     effect: NoSchedule
+#   - key: node-role.kubernetes.io/controlplane
+#     value: "true"
+#     effect: NoSchedule
+#   - key: node-role.kubernetes.io/etcd
+#     value: "true"
+#     effect: NoExecute
```

Comment:
I’d make the commented tolerations the default as I think this is the most common setup and will facilitate configuration for the users.